ci/openshift.yml

```diff
@@ -74,29 +79,30 @@ jobs:
         echo "APP_NAME=$(basename $PWD)" | tee -a $GITHUB_ENV
 
     - name: Determine tag
-      if: env.TAG == ''
+      if: env.TAGS == ''
       run: |
-        echo "TAG=${GITHUB_SHA::7}" | tee -a $GITHUB_ENV
+        echo "TAGS=${GITHUB_SHA::7}" | tee -a $GITHUB_ENV
 
     # https://github.com/redhat-actions/buildah-build#readme
     - name: Build from Dockerfile
-      uses: redhat-actions/buildah-build@v1
+      id: image-build
+      uses: redhat-actions/buildah-build@v2
       with:
         image: ${{ env.APP_NAME }}
-        tag: ${{ env.TAG }}
+        tags: ${{ env.TAGS }}
         # If you don't have a dockerfile, see:
-        # https://github.com/redhat-actions/buildah-build#building-from-scratch
+        # https://github.com/redhat-actions/buildah-build#inputs-for-build-without-dockerfile
```

Comment:
looks like we broke this link when we renamed building-from-scratch.

Can you add an anchor to the buildah readme so that doesn't happen again?

Reply:
Okay, will do.